examples/text_labels_and_annotations/titles_demo.py

```diff
@@ -16,3 +16,44 @@
 plt.title('Right Title', loc='right')
 
 plt.show()
+
+###########################################################################
+# The vertical position is automatically chosen to avoid decorators on the
```

Comment:
Is „decorators“ a term we commonly use? If not and because of the duplicate meaning with python decorators, I‘d just use „tick labels and axis labels“ instead.

Reply:
Well, its used in the `constrainedlayout_guide`, but I wrote that, so ;-)  But it should be "decorations".  

We need some word for this other than "ticks, labels, extra legends, annotations, etc etc". and I think "decorations" is a reasonable choice.